contracts/axelar-gateway/tests/testdata/call_contract.golden

```diff
@@ -1,3 +1,17 @@
-contract: Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHK3M)
-topics: (Symbol(contract_called), Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAITA4), String(ethereum), String(0x4EFE356BEDeCC817cb89B4E9b796dB8bC188DC59), BytesN<32>(86, 87, 13, 226, 135, 215, 60, 209, 203, 96, 146, 187, 143, 222, 230, 23, 57, 116, 149, 95, 222, 243, 69, 174, 87, 158, 233, 244, 117, 234, 116, 50))
-data: (Bytes(18, 52))
\ No newline at end of file
+ContractCalledEvent {
+    caller: Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAITA4),
+    destination_chain: String(ethereum),
+    destination_address: String(0x4EFE356BEDeCC817cb89B4E9b796dB8bC188DC59),
+    payload_hash: BytesN<32>(86, 87, 13, 226, 135, 215, 60, 209, 203, 96, 146, 187, 143, 222, 230, 23, 57, 116, 149, 95, 222, 243, 69, 174, 87, 158, 233, 244, 117, 234, 116, 50),
+    payload: Bytes(18, 52),
+}
+
+Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHK3M)
+
+contract_called {
+    #[topic] caller: Address,
+    #[topic] destination_chain: String,
+    #[topic] destination_address: String,
+    #[topic] payload_hash: BytesN < 32 >,
```

Comment:
due to how tokens work, it splits up brackets, but a bit tricky to handle without using `std` so keeping it for now